Rakefile

```diff
@@ -7,27 +7,31 @@ Rake::TestTask.new(:integration_test) do |t|
   t.libs << "test"
   t.libs << "lib"
   t.test_files = FileList['test/integration/**/*_test.rb']
+  t.warning = false
 end
 
 desc("Run integration tests that CANNOT be run in parallel")
 Rake::TestTask.new(:serial_integration_test) do |t|
   t.libs << "test"
   t.libs << "lib"
   t.test_files = FileList['test/integration-serial/**/*_test.rb']
+  t.warning = false
 end
 
 desc("Run unit tests")
 Rake::TestTask.new(:unit_test) do |t|
   t.libs << "test"
   t.libs << "lib"
   t.test_files = FileList['test/unit/**/*_test.rb']
+  t.warning = false
 end
 
 desc("Run cli tests")
 Rake::TestTask.new(:cli_test) do |t|
   t.libs << "test"
   t.libs << "lib"
   t.test_files = FileList['test/exe/**/*_test.rb']
+  t.warning = false
```

Comment:
I was really struggling to crawl through the test output with the warnings on. Happy to move this elsewhere though if there's more to do.

Reply:
What warnings were you seeing? When I ran the tests, the only output I got was the test suite summaries, and a large log output from a failed assertion upon the log string.

Reply:
I don't remember exactly, it's possible it was just in CI.

Reply:
Alrighty. I'll open a separate draft PR (targeting this one) with the original rakefile and I'll see if I can repro.